app/xds_api/utils/xds_utils.py

```diff
@@ -45,25 +45,35 @@ def metadata_to_target(metadata_JSON):
 
         for record in metadata_dict:
             if 'metadata' in record:
-                currObj = record['metadata']
-                meta = {}
+                metadatObj = record['metadata']
 
-                meta["id"] = record["unique_record_identifier"]
-                meta["metadata_key_hash"] = record["metadata_key_hash"]
-                currObj["meta"] = meta
-                result_list.append(currObj)
+                if 'Metadata_Ledger' in metadatObj:
+                    currObj = metadatObj['Metadata_Ledger']
+                    currObj["Supplemental_Ledger"] = \
```

Comment:
Do we need to check for the existence of this key as we did for `Metadata_Ledger` above?